### DISCUSSION TITLE
Missing images in result

### DISCUSSION THREAD

**Discussion by emryswizz** [searx/searx#3292] [General]
Hello,
In the result page, not all images are loaded... Any idea?

<img width="749" alt="CleanShot 2022-07-20 at 23 28 05@2x" src="https://user-images.githubusercontent.com/2249454/180123867-14ea422a-9f3c-4239-803c-b959ce6dde64.png">

**Comment by emryswizz** [comment]
Hello, any idea here on this issue?
Best.

**Reply by AlexBocken** [reply]
I have the same issue, so at least you're not alone. How is your searx set up? Morty + filtron and nginx as well? My guess is that there is something either in filtron (don't know much about it sadly) or in the reverse proxy set up in nginx that does not allow for images to get passed the whole way through
